test/node/server.go

```diff
@@ -75,6 +75,7 @@ func StartServer(t *testing.T, configFunc ...func(internalHttpServerURL, publicH
 	t.Setenv("NUTS_EVENTS_NATS_PORT", natsPort)
 	t.Setenv("NUTS_EVENTS_NATS_HOSTNAME", "localhost")
 	t.Setenv("NUTS_URL", publicHttpServerURL)
+	t.Setenv("NUTS_VDR_DIDMETHODS", "nuts")
```

Comment:
this looks off; the RTD configuration reference indicates `didmethods` is a root property; https://nuts-node.readthedocs.io/en/latest/pages/deployment/configuration.html

Reply:
#3284